core/domain/exp_services_test.py

```diff
@@ -1673,6 +1682,17 @@ class ExplorationCommitLogUnitTests(ExplorationServicesUnitTests):
         'post_commit_status': 'public'
     }
 
+    COMMIT_UPDATE_FIRST_PUBLISHED_MSEC_EXP_2 = {
```

Comment:
Is there another way to update activity rights besides rights_manager._save_activity_rights? If not, _save_activity_rights makes a commit to the exploration history whenever a change is made to any of the attributes in activity rights.